db/migrate/20220521164701_create_categories.rb

```diff
@@ -0,0 +1,10 @@
+class CreateCategories < ActiveRecord::Migration[6.1]
+  def change
+    create_table :categories do |t|
+      t.string :name
```

Comment:
Por convenciones de RoR es buena práctica tener la restrición tanto en la base de datos como en el modelo.
```suggestion
      t.string :name, null: false
```